.pre-commit-config.yaml

```diff
@@ -35,11 +35,10 @@ repos:
       - id: copyright-headers
         name: copyright-headers
         language: python
-        files: \.(c|h|cpp|hpp|cc|cxx|m|mm|inc|java)$
+        files: \.(c|h|cpp|hpp|cc|cxx|m|mm|java)$
```

Comment:
Does this mean `inc` files will be excluded from fixing headers by hooks? Why, if they also need fixing?

Reply:
I thought we weren't adding the copyright header to those, but it seems with do, with the exception of all `core/math/bvh_*.inc`. I can exclude those instead.

Reply:
Fixed!